### ISSUE TITLE
Autobrake doesn't disarm when manually braking

### ISSUE THREAD

**Issue by MCCVeen** [mSparks43/747-400#587]
I just landed with Autobrakes, and around 80 knots I started braking manually with the foot pedals but the autobrake didn't disarm. 

![image](https://user-images.githubusercontent.com/75425109/120510440-b821d580-c3c9-11eb-9ea7-782a8602e0c0.png)

**Comment by crazytimtimtim** [comment]
Wil fix :)

**Comment by MCCVeen** [comment]
> Wil fix :)

Thank you!!

**Comment by crazytimtimtim** [comment]
Wait... Isn't this already done? https://github.com/mSparks43/747-400/blob/95d57a3678f51b306ca4b2a445c76cff2a1ae5f3/plugins/xtlua/scripts/B747.10.xt.gear/B747.10.xt.gear.lua#L371

**Comment by MCCVeen** [comment]
> Wait... Isn't this already done?
> 
> https://github.com/mSparks43/747-400/blob/95d57a3678f51b306ca4b2a445c76cff2a1ae5f3/plugins/xtlua/scripts/B747.10.xt.gear/B747.10.xt.gear.lua#L371


Yeah it was implemented around new years but I think it broke or something. Maybe something to do with #258 ?

**Comment by crazytimtimtim** [comment]
> > Wait... Isn't this already done?
> > https://github.com/mSparks43/747-400/blob/95d57a3678f51b306ca4b2a445c76cff2a1ae5f3/plugins/xtlua/scripts/B747.10.xt.gear/B747.10.xt.gear.lua#L371
> 
> Yeah it was implemented around new years but I think it broke or something. Maybe something to do with #258 ?

I don't see why that would be messing up something, but it's possible. I did that separately from the code I linked above (this is done in gear, I did #258 fixes in FlightControls), so I think I'll try moving it to gear to see if that fixes it.

**Comment by MCCVeen** [comment]
> > > Wait... Isn't this already done?
> > > https://github.com/mSparks43/747-400/blob/95d57a3678f51b306ca4b2a445c76cff2a1ae5f3/plugins/xtlua/scripts/B747.10.xt.gear/B747.10.xt.gear.lua#L371
> > 
> > 
> > Yeah it was implemented around new years but I think it broke or something. Maybe something to do with #258 ?
> 
> I don't see why that would be messing up something, but it's possible. I did that separately from the code I linked above (this is done in gear, I did #258 fixes in FlightControls), so I think I'll try moving it to gear to see if that fixes it.

strange, keep me updated

**Comment by Matt726-S** [comment]
idk if this is what you’re talking about, but when i push my toe brakes the autobrake goes to disarm position as normal

**Comment by crazytimtimtim** [comment]
> idk if this is what you’re talking about, but when i push my toe brakes the autobrake goes to disarm position as normal

I believe so this seems to be working.

**Comment by MCCVeen** [comment]
> > idk if this is what you’re talking about, but when i push my toe brakes the autobrake goes to disarm position as normal
> 
> I believe so, this seems to be working.

Didn't work for me a few days ago with the latest update

**Comment by MCCVeen** [comment]
Just tested it, still doesn't work untill I put in thrust

**Comment by MCCVeen** [comment]
Sorry, I didn't explain this correctly. The AUTOBRAKES Eicas message doesn't display when the autobrakes are disarmed after thrust is put in @crazytimtimtim

**Comment by crazytimtimtim** [comment]
> 
> 
> Sorry, I didn't explain this correctly. The AUTOBRAKES Eicas message doesn't display when the autobrakes are disarmed, only after thrust is put in @crazytimtimtim

Ok, but it still disengages?

**Comment by crazytimtimtim** [comment]
This works, just tested it. I think this can be closed.

**Comment by MCCVeen** [comment]
> > Sorry, I didn't explain this correctly. The AUTOBRAKES Eicas message doesn't display when the autobrakes are disarmed, only after thrust is put in @crazytimtimtim
> 
> Ok, but it still disengages?

Sorry, didn't see this. Yes it disengages but the AUTOBRAKES Eicas message doesn't come up when the autobrake disarms, at least in the version I used a few weeks ago when I wrote this, and as far as I know no one made a fix for that yet

**Comment by crazytimtimtim** [comment]
Ok, I will take a look and likely fix.

**Comment by crazytimtimtim** [comment]
@MCCVeen Finally got back to working on the sparky, can you specify the AUTOBRAKES Eicas message you are talking about?

**Comment by crazytimtimtim** [comment]
Sorry about that, it is indeed fixed now. Will upload soon.
